packages/webapp/src/containers/Animals/Inventory/useAnimalInventory.tsx

```diff
@@ -229,6 +353,7 @@ const useAnimalInventory = () => {
     { label: 'customAnimalTypes', hook: useGetCustomAnimalTypesQuery },
     { label: 'defaultAnimalBreeds', hook: useGetDefaultAnimalBreedsQuery },
     { label: 'defaultAnimalTypes', hook: useGetDefaultAnimalTypesQuery },
+    { label: 'animalSexes', hook: useGetAnimalSexesQuery },
```

Comment:
Data not used by inventory, but I wanted it to get into the aggregate `isLoading` state.